persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -171,13 +169,12 @@ private BalSyntaxTreeGenerator() {}
     /**
      * method to read ballerina files.
      */
-    public static EntityMetaData getEntityRecord(Path filePath)
+    public static EntityMetaData getEntityRecord(SyntaxTree balSyntaxTree)
```

Comment:
```suggestion
    public static EntityMetaData getEntityMetadata(SyntaxTree balSyntaxTree)
```

Reply:
Updated in [c255915](https://github.com/ballerina-platform/persist-tools/pull/70/commits/c255915e8c07c686564fb7b9c6375f2af3507865)